### PR TITLE
Issue 641 - Use pat-modal instead of prepOverlay for Plone 5

### LINKED ISSUE #641: TypeError accessing Compose view in Plone 5
Plone 5.0.5 with collective.cover installed from master branch (1.2b2, unreleased).

After creating a cover object with Layout A, try to access the Compose tab; the following JavaScript error is shown in the console:

```javascript
21:37:23.653 TypeError: $(...).prepOverlay is not a function
<anonymous>compose.js:99
m.Callbacks/j()default.js:2
m.Callbacks/k.fireWith()default.js:2
.ready()default.js:2
J()default.js:2
1compose.js:99:3
```

### PR DESCRIPTION
**Pull request by rodfersou**
closes #641

### CODE REVIEW

File: src/collective/cover/static/js/compose.js [hunk 1]
```diff
@@ -96,117 +96,119 @@ $(document).ready(function() {
...
-    filter: '.tile-content',
-    formselector: '#edit_tile',
-    closeselector: '[name="buttons.cancel"]',
-    noform: 'close',
-    beforepost: function(return_value, data_parent) {
-      // Before post data, populate the textarea (textarea.mce_editable) with the contents of  iframe created by TinyMCE call.
-      // TODO: This does not solves, if we have more of a textarea widget in tiles. What's the solution?
-      var iframes = jQuery('#edit_tile iframe');
-      var mcs = {};
+  if ($.fn.prepOverlay !== undefined) {
```
Comment (rodfersou) [inline src/collective/cover/static/js/compose.js]: I just reindent the code inside the ``if`` statement

**Comment by hvelarde** [comment]
@rodfersou we'll need to create an upgrade step before the release to address this.

**Comment by rodfersou** [comment]
@hvelarde I think this PR was not ready

**Comment by hvelarde** [comment]
what was missing?

**Comment by rodfersou** [comment]
I was going to check when there is p.a.widgets and prepOverlay.

Also I suspect that tile rich text configuration will not work with this implementation.

**Comment by hvelarde** [comment]
you can do that as part of #667.

**Pull request merged**